lib/embulk/input/google_analytics/client.rb

```diff
@@ -108,6 +108,7 @@ def time_parse_with_profile_timezone(time_string)
           unless parts
             # strptime was failed. Google API returns unexpected date string.
             Embulk.logger.warn("Failed to parse #{task["time_series"]} data. The value is '#{time_string}'(#{time_string.class}) and it doesn't match with '#{date_format}'.")
+            raise Embulk::DataError.new("Can't parse raw time returned from Google API, raw time is #{time_string}, expected format #{date_format}")
```

Comment:
Let's just replace `Embulk.logger.warn` log with `raise Embulk::DataError.new` (no need for another error message).